cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -177,19 +175,6 @@ setupTrace loggingLayer = do
       hn0 <- pack <$> getHostName
       return $ take 8 $ fst $ breakOn "." hn0
 
--- | The node sends its real up time, every second.
-traceNodeUpTime
-  :: Trace IO Text
-  -> Word64
-  -> IO ()
-traceNodeUpTime tr nodeLaunchTime = do
-  now <- getMonotonicTimeNSec
-  let upTimeInNs = now - nodeLaunchTime
-  meta <- mkLOMeta Notice Public
-  traceNamedObject tr (meta, LogValue "upTime" (Nanoseconds upTimeInNs))
-  threadDelay 1000000
-  traceNodeUpTime tr nodeLaunchTime
-
```

Comment:
Why is this removed? I suppose because you're now tracing `nodeStartTime`? Maybe make this unrelated change in a separate commit explaining why instead of sneaking it in here.

Reply:
Fixed. It will be removed in a separate PR.